install/collabora-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: remz1337
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Collabora Online"
+cd /usr/share/keyrings
+$STD wget -q https://collaboraoffice.com/downloads/gpg/collaboraonline-release-keyring.gpg
+cd ~
+cat <<EOF >/etc/apt/sources.list.d/collaboraonline.sources
+Types: deb
+URIs: https://www.collaboraoffice.com/repos/CollaboraOnline/CODE-deb
+Suites: ./
+Signed-By: /usr/share/keyrings/collaboraonline-release-keyring.gpg
+EOF
+$STD apt update 
+$STD apt install -y coolwsd code-brand collaboraoffice*
+sed -i 's|<enable type="bool" desc="Controls whether SSL encryption between coolwsd and the network is enabled (do not disable for production deployment). If default is false, must first be compiled with SSL support to enable." default="true">true</enable>|<enable type="bool" desc="Controls whether SSL encryption between coolwsd and the network is enabled (do not disable for production deployment). If default is false, must first be compiled with SSL support to enable." default="true">false</enable>|g' /etc/coolwsd/coolwsd.xml
```

Comment:
is the `desc` attribute required by the COOLWS server?

Reply:
Not sure if required per se (we can test it out), but that's what in the file by default so I would be weary of removing it. My guess is that the `desc` attribute is used in some form of UI.

If your concern is that this line is too long, I can try to optimize using wildcards.